index.html

```diff
@@ -555,21 +555,21 @@ <h4>Prove Presentation</h4>
       </section>
 
       <section>
-        <h4>Presentation Availability</h4>
+        <h4>Initiate Exchange</h4>
         <p>
         </p>
 
         <div class="api-detail"
-          data-api-endpoint="post /presentations/available"></div>
+          data-api-endpoint="post /exchanges/{exchange-id}"></div>
       </section>
 
       <section>
-        <h4>Submit Presentation</h4>
+        <h4>Continue Exchange</h4>
         <p>
         </p>
 
         <div class="api-detail"
-          data-api-endpoint="post /presentations/submissions"></div>
+          data-api-endpoint="post /exchanges/{exchange-id}/{transaction-uuid}"></div>
```

Comment:
I believe this needs to be updated to the put route

Reply:
Done in a172d8118a70ae6d42e8eadd333727b151d6b4a3.